src/test/java/api/loans/scenarios/CheckInWithdrawnItemTest.java

```diff
@@ -0,0 +1,112 @@
+package api.loans.scenarios;
+
+import static api.support.matchers.JsonObjectMatcher.hasJsonPath;
+import static com.jayway.jsonpath.matchers.JsonPathMatchers.hasNoJsonPath;
+import static org.hamcrest.CoreMatchers.allOf;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import org.folio.circulation.support.http.client.IndividualResource;
+import org.junit.Test;
+
+import api.support.APITests;
+import api.support.CheckInByBarcodeResponse;
+import api.support.builders.ItemBuilder;
+import api.support.builders.RequestBuilder;
+import api.support.http.InventoryItemResource;
+import io.vertx.core.json.JsonObject;
+
+public class CheckInWithdrawnItemTest extends APITests {
+  @Test
+  public void canCheckInAtHomeLocation() {
+    final InventoryItemResource item = itemsFixture
+      .basedUponSmallAngryPlanet(ItemBuilder::withdrawn);
+
+    assertThat(item.getJson(), hasJsonPath("status.name", "Withdrawn"));
+
+    final CheckInByBarcodeResponse response = loansFixture.checkInByBarcode(item);
+
+    assertThat(response.getJson(), allOf(
+      hasNoJsonPath("loan"),
+      hasJsonPath("item.status.name", "Available")
+    ));
+
+    assertThat(itemsClient.getById(item.getId()).getJson(),
+      hasJsonPath("status.name", "Available"));
```

Comment:
Could the magic values for the item status be extracted into constants that all of the tests could use?